controller_report_xls/controllers/main.py

```diff
@@ -30,8 +30,8 @@
 import xlwt
 from cssutils import parseString
 from lxml import etree
-from openerp.addons.report.controllers import main
-from openerp.addons.web.http import request, route
+from odoo.addons.report.controllers import main
+from odoo.addons.web.http import request, route
```

Comment:
This is an "issue" coming from the previous version, it therefore could be fixed there. This is not strictly part of the migration.